x-pack/solutions/observability/plugins/observability/public/services/dashboard_service.test.ts

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This test should be deleted

Reply:
I removed it in this [commit](https://github.com/elastic/kibana/pull/209668/commits/323d38accd86ee0d9b313d17a2051e71a02f4c4e).